src/seeds.history.cpp

```diff
@@ -113,202 +126,236 @@ void history::trxentry(name from, name to, asset quantity) {
     return;
   }
 
-  bool from_is_organization = from_user->type == "organisation"_n;
+  uint64_t day = utils::get_beginning_of_day_in_seconds();
+  daily_transactions_tables transactions(get_self(), day);
+
+  uint64_t transaction_id = transactions.available_primary_key();
+  uint64_t timestamp = eosio::current_time_point().sec_since_epoch();
+
+  bool from_is_organization = from_user -> type == "organisation"_n;
+  bool to_is_organization = to_user -> type == "organisation"_n;
+
+  int64_t transactions_cap = int64_t(config_get("qev.trx.cap"_n));
+  int64_t max_transaction_points_individuals = int64_t(config_get("i.trx.max"_n));
+  int64_t max_transaction_points_organizations = int64_t(config_get("org.trx.max"_n));
+
+  double from_trx_multiplier = (
+    from_is_organization ? 
+    std::min(max_transaction_points_organizations, quantity.amount) : 
+    std::min(max_transaction_points_individuals, quantity.amount)
+  ) / 10000.0;
   
-  if (from_is_organization) {
-    org_tx_tables orgtx(get_self(), from.value);
-    orgtx.emplace(_self, [&](auto& item) {
-      item.id = orgtx.available_primary_key();
-      item.other = to;
-      item.in = false;
-      item.quantity = quantity;
-      item.timestamp = eosio::current_time_point().sec_since_epoch();
+  double to_trx_multiplier = std::min(max_transaction_points_organizations, quantity.amount) / 10000.0;
+
+  transactions.emplace(_self, [&](auto & transaction){
+    transaction.id = transaction_id;
+    transaction.from = from;
+    transaction.to = to;
+    transaction.volume = quantity.amount;
+    transaction.qualifying_volume = std::min(transactions_cap, quantity.amount);
+    transaction.from_points = uint64_t(ceil(from_trx_multiplier * utils::get_rep_multiplier(to)));
+    transaction.to_points = uint64_t(ceil(to_trx_multiplier * utils::get_rep_multiplier(from)));
```

Comment:
I have a question on this
- if "to" is an individual then transaction.to_points  should be == 0 (as individuals do not receive points on incoming transactions)

Reply:
Oh yeah, that's right I am not setting them to 0, although the field "to" is not being used to calculate transaction points for individuals, but yeah I will change it.